.github/workflows/publishBetaVersion.yml

```diff
@@ -6,7 +6,7 @@ on:
 jobs:
   test:
     name: Publish beta version
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
Revert to ubuntu

Reply:
@shashankraj0202 This causes issues with the pipelines since locally the files created in mac are different.

Reply:
okay